examples/cli.rs

```diff
@@ -422,7 +423,29 @@ async fn run<C: Store + MessageStore>(subcommand: Cmd, config_store: C) -> anyho
         Cmd::Block => unimplemented!(),
         Cmd::Unblock => unimplemented!(),
         Cmd::UpdateContact => unimplemented!(),
-        Cmd::ListGroups => unimplemented!(),
+        Cmd::ListGroups => {
+            let manager = Manager::load_registered(config_store)?;
+            let mut groups = manager.get_groups()?.peekable();
+            if groups.peek().is_some() {
```

Comment:
I get that you want to print "No groups found" but I would just skip the `peekable` and test entirely. If it prints nothing, then it's empty.

Reply:
i removed the "No groups found"